src/main/scala/org/broadinstitute/dsde/firecloud/FireCloudApiService.scala

```diff
@@ -33,6 +34,12 @@ object FireCloudApiService {
       case withErrorReport: FireCloudExceptionWithErrorReport =>
         complete(withErrorReport.errorReport.statusCode.getOrElse(StatusCodes.InternalServerError) -> withErrorReport.errorReport)
       case e: Throwable =>
+        // so we don't log the error twice when debug is enabled
+        if (logger.underlying.isDebugEnabled) {
+          logger.debug(e.getMessage, e)
```

Comment:
Same comment as in the other PR https://github.com/broadinstitute/rawls/pull/1316/files#r526327849